### DISCUSSION TITLE
The upload progress cannot be updated when using q-uploader

### DISCUSSION THREAD

**Discussion by tlds119** [quasarframework/quasar#10040] [General - Components / Directives / etc]
When I uploaded the file, there was a problem. Now the file can be uploaded normally, and the parameters received by springboot are correct, but the upload progress cannot be updated, and handleuploaded, handleuploading, handlefailed are not executed. Who can help me?

[q-uploader code.txt](https://github.com/quasarframework/quasar/files/6796598/q-uploader.code.txt)
![capture](https://user-images.githubusercontent.com/4993767/125189184-63e00000-e269-11eb-810f-3fd0ccfdb86c.PNG)
![capture2](https://user-images.githubusercontent.com/4993767/125189188-6b070e00-e269-11eb-9e7f-abac56a2449c.PNG)

**Comment by Griefed** [comment]
I'd like to know how to update the progress percentage as well.
Funnily enough, the LoadingBar get's update accordingly, but I have no clue how to update the percentage of the Q-Uploader
![image](https://user-images.githubusercontent.com/44273438/152963875-6bb5643a-89bd-4030-a5e4-6a17a871d557.png)

**Comment by Griefed** [comment]
My problem was that I was using a factory incorrectly.

Heres an example for ZIP-archives:

#### Q-Uploader configuration:

```vue
<q-uploader
    class="full-width"
    url="/api/v1/zip/upload"
    method="POST"
    field-name="file"
    auto-upload
    label="Only ZIP-archives (max 500Mb)"
    accept=".zip,application/zip"
    max-file-size="512000000"
/>
```

#### Spring Boot upload Method:

*Class has mapping to `api/v1/zip`*

```java
@PostMapping("/upload")
public String handleFileUpload(@RequestParam("file") final MultipartFile file) throws IOException {
    final byte[] fileBytes = uploadedFile.getBytes();

    final Path zipPath = Paths.get(
        new File("./work/modpacks/" + uploadedFile.getOriginalFilename())
            .getAbsolutePath().replace("\\","/"));

    Files.write(zipPath,fileBytes);

    return zipPath;
}
```

Everything is displayed correctly:
![image](https://user-images.githubusercontent.com/44273438/152976914-1dd20bde-56bc-4ecc-8647-fe86801e7757.png)
